### PR TITLE
Update kokoro to pull latest version of cmake.

### LINKED ISSUE #334: Kokoro build needs current cmake (currently has a 2016 version)
The default cmake in the kokoro build is from 2016 (3.7.1).

We now need at least cmake 3.13 so Kokoro builds fail
https://source.cloud.google.com/results/invocations/d889a6fc-c696-496f-b848-05c54dd78569/targets/foss-fpga-tools%2Falainmarcel-surelog%2Fpresubmit/log

So we need do do something similar to the equivalent travis change
https://github.com/alainmarcel/Surelog/pull/333/files

### PR DESCRIPTION
**Pull request by hzeller**
Fixes #334

Signed-off-by: Henner Zeller <h.zeller@acm.org>

### CODE REVIEW

File: .github/kokoro/steps/hostsetup.sh [hunk 1]
```diff
@@ -7,6 +7,9 @@ echo "========================================"
 echo "Host adding PPAs"
 echo "----------------------------------------"
 sudo apt-add-repository 'ppa:ubuntu-toolchain-r/test'
+sudo add-apt-repository 'deb https://apt.kitware.com/ubuntu/ xenial main'
```
Comment (mithro) [inline .github/kokoro/steps/hostsetup.sh:10]: https://github.com/SymbiFlow/prjxray/blob/3749dc63c490e0064f243d4289e50ff72cbd83ad/.github/kokoro/steps/hostsetup.sh#L12-L18
```
echo
echo "========================================"
echo "Host adding PPAs"
echo "----------------------------------------"
wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | sudo apt-key add -
sudo apt-add-repository 'deb https://apt.kitware.com/ubuntu/ xenial main'
echo "----------------------------------------"
```

Comment (hzeller) [reply .github/kokoro/steps/hostsetup.sh:11]: thanks Tim, this looks nicer indeed. Updated.

**Comment by alaindargelas** [comment]
Can you update the INSTALL.md with what are the steps now to install Surelog?

I don't think this works anymore:

sudo apt-get install build-essential cmake git pkg-config tclsh swig uuid-dev libgoogle-perftools-dev python3 python3-dev

**Comment by hzeller** [comment]
The configuration in INSTALL.md should still work with a system younger than 2 years. Did you run into any trouble @alaindargelas ?

The thing I have to address in the travis and kokoro instances is that these compile system are  by default at a state of around 2015...

**Comment by hzeller** [comment]
Alright, ready to be merged.

**Comment by alaindargelas** [comment]
Ubuntu still has an older cmake I think

**Pull request merged**